### DISCUSSION TITLE
403 forbidden response when trying to download or fetch as asset via API

### DISCUSSION THREAD

**Discussion by SharadKumar** [directus/directus#17318] [Archive]
### Describe the Bug

I am running `9.22.3`
<img width="276" alt="image" src="https://user-images.githubusercontent.com/5488144/215071686-56a31a1d-908e-476e-ab9b-7599ce73244a.png">

I have set up few collections and am using snapshot from time to time to be able to have DX of config in code. I have also setup AWS `S3` for files storage. Everythig is working as expected on the App, but when I try to download the file, it throws `403` Forbidden error. When I try to access the direct link (taken from WYSIWYG source view), it has same outcome! Base64 view of same file in the App is showing the image fine. File is ofcourse showing fine in S3 bucket as well.

### To Reproduce

Not sure. What I have is:

- Directus with S3 configured. I do not have `STORAGE_S3_ENDPOINT` var, but have all other vars in place. Didn't think it is needed, as everything worked previously.
- Created few folders in Files library to map each with a collection.
- Added media asset to a WYSIWYG body field in one of the collection. File got uploaded but failed to fetch. I tried the generated asset url directly, no result. I ensured `Public` role is allowed `Read` permission on the collection.

### Errors Shown

File is showing fine in App:
<img width="636" alt="image" src="https://user-images.githubusercontent.com/5488144/215072872-3e79080f-15e5-41ff-80e5-236e0d487f1e.png">

Tried to download the image in Chrome:
<img width="255" alt="image" src="https://user-images.githubusercontent.com/5488144/215072634-6cf6c478-4f68-40ec-a212-2dba1a482eb4.png">

WYSIWYG view:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/5488144/215073009-806ef251-0b46-48fd-ab11-1bdc0bb7cd12.png">

Then when I access via API, I get:
```
{"errors":[{"message":"You don't have permission to access this.","extensions":{"code":"FORBIDDEN"}}]}
```
<img width="820" alt="image" src="https://user-images.githubusercontent.com/5488144/215073140-f34230ee-2f18-4066-89b4-7015c1e6023f.png">

### What version of Directus are you using?

9.22.3

### What version of Node.js are you using?

16.19.0

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

Local

**Comment by paescuj** [comment]
Converted to a discussion since this is no issue.

You need to configure a static token for the WYSIWYG field in order to be able to see inlined images:
<img width="500" src="https://user-images.githubusercontent.com/5363448/215077134-a26b98e7-4ba4-476d-99c7-295b761e3e19.png">

See also https://github.com/directus/directus/discussions/9831#discussioncomment-1868772 for an explanation / example how to use the static token.

**Reply by SharadKumar** [reply]
Many thanks @paescuj
